site/content/docs/user/quick-start.md

```diff
@@ -151,6 +151,13 @@ kind load docker-image my-custom-image:unique-tag
 kubectl apply -f my-manifest-using-my-image:unique-tag
 ```
 
+**Note**: You can get a list of images present on a cluster node by
+using `docker exec`:
+```
+docker exec -it my-node-name crictl images
+```
+Where `my-node-name` is the name of the Docker container.
```

Comment:
we should mention `kind get nodes` or `kubectl get nodes` and specifically that we're talking about a node container

Reply:
can be done in a follow up.